hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -80,7 +82,8 @@ public class MerkleNetworkContext extends AbstractMerkleLeaf {
 	static final int RELEASE_0200_VERSION = 6;
 	static final int RELEASE_0240_VERSION = 7;
```

Comment:
Do we need migration for these two versions here and `getMinimumSupportdVersion` should change?

Reply:
Bumped min supported to 0.24.x